### PR TITLE
Hosting: Add contextual inline help

### LINKED ISSUE #37899: Hosting: Link to related support docs from help
We should update help in /hosting-config to open to newly added support docs.

Screen Shot 2019-11-22 at 2 40 38 PM

### PR DESCRIPTION
**Pull request by glendaviesnz**
#### Changes proposed in this Pull Request

* Adds some contextual inline help links for the new hosting section

#### Testing instructions

* Checkout this PR and open Hosting Configuration Section for an eligible Atomic site
* Click on the inline help button at bottom right
* Check that the help popup shows links for SFTP Credential and Database Access

Before:
Screen Shot 2019-11-25 at 2 21 28 PM

After:
Screen Shot 2019-11-25 at 2 15 57 PM

Fixes #37899

### CODE REVIEW

File: client/blocks/inline-help/contextual-help.js [hunk 1]
```diff
@@ -905,6 +905,24 @@ const contextLinksForSection = {
 			),
 		},
 	],
+	hosting: [
+		{
+			link: localizeUrl( 'https://en.support.wordpress.com/sftp/' ),
```
Comment (glendaviesnz) [inline client/blocks/inline-help/contextual-help.js]: I couldn't find the actual slugs for these two links - took post_ids from one of Ian's posts, but can't access the actual pages - I probably don't have editor access to the help pages, so if someone with access can verify the two post_ids and slugs and let me know, or update yourself, please :-)

File: client/blocks/inline-help/contextual-help.js [hunk 2]
```diff
@@ -905,6 +905,24 @@ const contextLinksForSection = {
...
+			description: translate(
+				"Access and edit your website's files directly by using an SFTP client."
+			),
+		},
+		{
+			link: localizeUrl( 'https://en.support.wordpress.com/database-access/' ),
+			post_id: 159822,
+			title: translate( 'Database Access' ),
+			description: translate(
+				'For the tech-savvy, manage your database with phpMyAdmin and run a wide range of operations with MySQL.'
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **14** times:
`translate( 'Manage your database with phpMyAdmin and run a wide range of operations with MySQL.' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/hosting-section-inline-help)

File: client/blocks/inline-help/contextual-help.js (hunk 1 above)
Comment (mmtr) [reply client/blocks/inline-help/contextual-help.js]: I have access, so I updated the slugs with actual permalinks (post ids were correct). I also changed the titles so they mache the actual support page title (seems to be the pattern with other pages).

**Comment by automattic-ian** [comment]
@mmtr Those descriptions are GREAT.

**Comment by gwwar** [comment]
Let's also add https://en.support.wordpress.com/php-version-switching/ and I think we may want to feature flag this. Will confirm in testing

**Comment by glendaviesnz** [comment]
> Let's also add https://en.support.wordpress.com/php-version-switching/

@gwwar are you able to give me the post_id for that page please? - I don't seem to have admin access to the support docs.

**Comment by automattic-ian** [comment]
@glendaviesnz it should be 160597

**Comment by gwwar** [comment]
The permalinks should work, the version switching page is already published so that should be a good one to test.

**Comment by glendaviesnz** [comment]
Add php version change docs and also limited sftp/database links to 'hosting/sftp-phpmyadmin' feature flag

'hosting/sftp-phpmyadmin' = true:

Screen Shot 2019-11-26 at 10 34 49 AM

Screen Shot 2019-11-26 at 10 30 34 AM

'hosting/sftp-phpmyadmin' = false:

**Pull request merged**